spanner/integration_test.go

```diff
@@ -1699,6 +1718,22 @@ func TestIntegration_BasicTypes(t *testing.T) {
 		{col: "NumericArray", val: []NullNumeric{{n1, true}, {n2, true}, {}}},
 	}
 
+	if !isEmulatorEnvSet() {
+		tests = append(tests, []struct {
+			col  string
+			val  interface{}
+			want interface{}
+		}{
+			{col: "JSON", val: NullJSON{msg, true}, want: msg},
+			{col: "JSON", val: NullJSON{msg, true}, want: NullJSON{unmarshalledJSONstruct, true}},
+			{col: "JSON", val: NullJSON{msg, false}},
+			{col: "JSON", val: nil, want: NullJSON{}},
+			{col: "JSONArray", val: []NullJSON(nil)},
+			{col: "JSONArray", val: []NullJSON{}},
+			{col: "JSONArray", val: []NullJSON{{msg, true}, {msg, true}, {}}},
```

Comment:
Do we support decoding an `ARRAY<JSON>` to a `[]Struct`, as we do support decoding a `JSON` to a `Struct`? If so, could we add a test for that here as well?

Reply:
As I updated to not support a `JSON` to a `Struct`, this will be not necessary any more.